### ISSUE TITLE
[Content item horizontal with Media] Tab focus is going to the video player but it is not visible at all

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#6510]
## Detailed description

Issue:
Tab focus is going to the video player but it is not visible at all.

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/5a75040b-64e8-4af2-a24c-7af19bf215a5)

**Comment by emyarod** [comment]
@Praveen-sr it looks like you have the image viewer in the screenshot, and when the storybook "media type" knob is changed to `video`, the video is accessible via keyboard and visible to the user. can you clarify what the original issue is?

**Comment by Praveen-sr** [comment]
@emyarod The issue is with the video. i.e.  when the storybook "media type" knob is changed to video.
This defect is reproducible in web component staging but working fine in web component canary.

Note: Accessibility testing is on Firefox browser.

**Comment by emyarod** [comment]
@Praveen-sr I can tab focus on the video player and the player is visible in Firefox (web components stage) as well unless I am misunderstanding

![image](https://user-images.githubusercontent.com/8265238/133463628-88f10643-1c8e-47f7-9b4d-7aa739d1b416.png)

but given that the issue you reported doesn't seem to be present in canary, would it be safe to call it resolved?

**Comment by Praveen-sr** [comment]
@emyarod Yes, its working fine in web component canary. On Firefox 

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/03cf878d-dfc1-4178-b17d-3ebc14263af3)

**Comment by RobertaJHahn** [comment]
Closing based on Praveen's comments above.
